api/app/signals/apps/email_integrations/services.py

```diff
@@ -51,7 +53,7 @@ def status_mail(cls, signal, dry_run=False) -> bool:
         return False
 
     @classmethod
-    def system_mail(cls, signal: Union[str, Signal], action_name: str, dry_run=False, **kwargs) -> bool:
+    def system_mail(cls, signal: Union[str, Signal], action_name: str, dry_run=False, recipient_list=[], **kwargs) -> bool:
```

Comment:
Remove the recipient_list here and add this functionality to the `AssignedAction`

Reply:
Done 👍